packages/plugins/i18n/admin/src/components/CMEditViewLocalePicker.tsx

```diff
@@ -83,12 +83,12 @@ const CMEditViewLocalePicker = ({
     }
 
     if (status === 'did-not-create-locale') {
-      push({
+      navigate({
```

Comment:
I wonder if you could use a relative pathname instead?

Reply:
I tried this, I think how the routes are set up at the moment it doesn't work as well because a relative path adds `create` on, doing `../create` takes away too much. Something to look at in the near future though for sure.